pkg/hintrunner/zero/zerohint_signature.go

```diff
@@ -88,6 +88,86 @@ func createVerifyZeroHinter(resolver hintReferenceResolver) (hinter.Hinter, erro
 	return newVerifyZeroHint(val, q), nil
 }
 
+// VerifyZeroV3 hint verifies that a packed value is zero modulo the Curve25519 prime
+// and stores in memory the quotient of the modular divison of the packed value by
+// Curve25519 prime
+//
+// `newVerifyZeroV3Hint` takes 2 operanders as arguments
+//   - `value` is the value that will be verified
+//   - `q` is the variable that will store the quotient of the modular division
+//
+// `newVerifyZeroV3Hint` writes the quotient of the modular division of the packed value
+// by Curve25519 prime to the memory address corresponding to `q`
+func newVerifyZeroV3Hint(val, q hinter.ResOperander) hinter.Hinter {
```

Comment:
some unit tests for this?

Reply:
well VerifyZeroV2 and VerifyZeroV3 are actually stricly the same as verifyZero which is already tested... i found it useless to add unit test given the already present unit test for verifyZero and the integration tests for all

what do you think?

Reply:
okok, I got it. Yeah I agree that if they copy the functionalities, then thats not worth adding a big unit test suite. In my draft PR on expanded uint256 div with remainder I had the same situation, so i just took the most sophisticated unit test from the base hint (which I modified) so that we stick to the convention of at least some unit testing per hint for the future reference